pkg/metallb/frrnodestate_test.go

```diff
@@ -0,0 +1,189 @@
+package metallb
+
+import (
+	"testing"
+
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/schemes/metallb/frrtypes"
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+var (
+	defaultNodeName           = "worker-0"
+	defaultFrrNodeStateNsName = "test-namespace"
```

Comment:
I don't think you need this as this seems to be a cluster-scoped resource

Reply:
removed